manageiq-gems-pending.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "activerecord",            ">= 5.0", "< 5.2"
   s.add_runtime_dependency "activesupport",           ">= 5.0", "< 5.2"
   s.add_runtime_dependency "awesome_spawn",           "~> 1.4"
-  s.add_runtime_dependency "aws-sdk",                 "~> 2.9.7"
+  s.add_runtime_dependency "aws-sdk",                 "~> 3"
```

Comment:
Please spell out the full gem version, otherwise we run the risk of accidentally pulling in v4 when it's released.

Reply:
done